utils/frame/try-runtime/cli/Cargo.toml

```diff
@@ -17,7 +17,7 @@ clap = { version = "3.1.6", features = ["derive"] }
 log = "0.4.8"
 parity-scale-codec = "3.0.0"
 serde = "1.0.136"
-zstd = "0.9.0"
+zstd = "0.10.0"
```

Comment:
while we're at it, I'd suggest using `default-features = false` here and elsewhere like in `primitives/maybe-compressed-blob/Cargo.toml`